cmake/CommonDefs.cmake

```diff
@@ -80,11 +72,11 @@ else()
 endif()
 endmacro(set_linking_type)
 
-# 
+#
 # Dependencies
 #
 
-# ALPSCore - brings MPI, HDF5, boost 
+# ALPSCore - brings MPI, HDF5, boost
 macro(add_alpscore)
 find_package(ALPSCore REQUIRED COMPONENTS hdf5 accumulators mc params)
```

Comment:
mc looks redundant here. I am not sure if alpscore::accumulators still exist. Do we actually need the dependency at all? Looking at the amount of code we use (https://github.com/aeantipov/pomerol/search?q=alpscore) - it's an absolute minimum. Feel free to address this in a separate PR if needed.

Reply:
> I am not sure if alpscore::accumulators still exist.

FYI, they do exist in the deprecated status.
New codes are meant to depend on [alea](https://github.com/ALPSCore/ALPSCore/tree/master/alea) instead (see also [this preprint](https://arxiv.org/abs/1811.08331), which has been under peer review for almost 2 years now).

> Do we actually need the dependency at all?

No, we do not. `add_alpscore()` is defined here but never called.
`https://github.com/aeantipov/pomerol/search?q=alpscore` finds only comments.

I'd rather remove this macro altogether and keep only that mention of ALPSCore in `README.md`.

Reply:
Let's remove it.